zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -1670,7 +1670,9 @@ dynssl.options.name              = Dynamic SSL Certificates
 dynssl.text.createnow            = Go to options panel and create certificate now.
 dynssl.text.notnow               = Not now, but create certificate later.
 dynssl.text.sslwontwork          = SSL won't work if you haven't created and imported an OWASP ZAP CA root certificate. You can create such a certificate any time in the options menu, so you do not have to create it right now.
-dynssl.warn.cert.expired = ZAPs Root CA certificate has expired as of {0} (now: {1}).\nYou should regenerate it and re-install it in your browsers.
+dynssl.warn.cert.expired = ZAPs Root CA certificate has expired as of {0} (now: {1}).\nYou should regenerate it and re-install it in your browsers.\n\
+Recreate the certificate and go to the relevant options page now?
```

Comment:
```suggestion
Recreate the certificate and go to the relevant options screen now?
```

Or `dialog`

Reply:
Changed to `Regenerate the certificate and go to the relevant options screen now?` ie change to "Regenerate" as well for consistancy